apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -934,6 +934,7 @@ export class ApiModelGenerator {
 
       parentApiItem.addMember(apiTypeAlias);
     }
+    this._processChildDeclarations(astDeclaration, exportedName, apiTypeAlias);
```

Comment:
Based on the comments in #3002 by @octogonz this this just need to be guarded like this for the time being?

```ts
  if (
      ts.isTypeAliasDeclaration(astDeclaration.declaration) &&
      ts.isTypeLiteralNode(astDeclaration.declaration.type)
    ) {
      this._processChildDeclarations(astDeclaration, exportedName, apiTypeAlias);
    }
```